.github/workflows/pr.yml

```diff
@@ -12,7 +12,7 @@ on:
       - release/*
 
 env:
-  ENTANDO_OPT_USE_PPL_TAG: "v1.4.0"
+  ENTANDO_OPT_USE_PPL_TAG: "v1.4.1"
```

Comment:
We need to update also post-merge.yml and ga-publication.yml

Reply:
it's correct, in develop we already have 1.4.0